python/cudf_polars/cudf_polars/containers/column.py

```diff
@@ -33,10 +34,32 @@ def __init__(
         null_order: plc.types.NullOrder = plc.types.NullOrder.BEFORE,
     ):
         self.obj = column
+        self.is_scalar = self.obj.size() == 1
+        if self.obj.size() <= 1:
+            is_sorted = plc.types.Sorted.YES
         self.is_sorted = is_sorted
         self.order = order
         self.null_order = null_order
 
+    @functools.cached_property
+    def obj_scalar(self) -> plc.Scalar:
+        """
+        A copy of the column object as a pylibcudf Scalar.
+
+        Returns
+        -------
+        pylibcudf Scalar object.
+
+        Raises
+        ------
+        RuntimeError if the column is not length-1.
+        """
+        if not self.is_scalar:
+            raise RuntimeError(
```

Comment:
Probably a ValueError (arguably a TypeError, but that's a harder argument IMO).
```suggestion
            raise ValueError(
```

Reply:
Done.